init.lua

```diff
@@ -60,6 +60,29 @@ local source = string.sub(debug.getinfo(1,'S').source, 2)
 local path = string.sub(source, 1, string.find(source, "/[^/]*$"))
 local noicon = path .. "noicon.png"
 
+local function cycle(altTabTable, altTabIndex, dir)
```

Comment:
If I move the cycle function to its original position, I either have to 
* make it global instead of local
* or foward declare it, i.e.
`local cycle` at the position where I put it and `cycle = function(...)` and the function body at the original position

I prefer the second variant. If you agree, then I will implement it this way.